synapse/lib/stormtypes.py

```diff
@@ -6249,13 +6343,17 @@ class CronJob(Prim):
                        'desc': 'A dictionary containing structured data about a cronjob for display purposes.', }}},
     )
     _storm_typename = 'storm:cronjob'
+    _ismutable = False
 
     def __init__(self, runt, cdef, path=None):
         Prim.__init__(self, cdef, path=path)
         self.runt = runt
         self.locls.update(self.getObjLocals())
         self.locls['iden'] = self.valu.get('iden')
 
+    def __hash__(self):
+        return hash(self.locls['iden'])
```

Comment:
this should hash the tuple of typename and iden since iden could be externally controlled/created